02.calendar/cal.rb

```diff
@@ -0,0 +1,34 @@
+#!/Users/heureux/.rbenv/versions/2.6.5/bin/ruby
```

Comment:
```suggestion
#!/usr/bin/env ruby
```

rbenvを使っている場合、shebangは上のように書くと良いみたいです。

https://yami-beta.hateblo.jp/entry/2013/05/14/203600

Reply:
ありがとうございます。最初「#!/usr/bin/env ruby」だと動かなかったのですが、
https://bootcamp.fjord.jp/reports/40855のくだりで`~/.bash_profile`を修正したのですが
その後である今だと確かに動きましたので修正しました！